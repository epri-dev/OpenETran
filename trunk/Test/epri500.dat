* incoming backflash for the EPRI 500-kV training example

time 1 6e-9 40e-6

* network definition

span 1
cable 1 355.0 3.0e8 -371.0e3
end
* end

* incoming line with surge
line  1  2 1 914.0 0 0
* ring bus
line  2  3 1  20.0 0 0
line  3  4 1  15.0 0 0
line  4  5 1  15.0 0 0
line  5  6 1  30.0 0 0
line  6  7 1  15.0 0 0
line  7  8 1  15.0 0 0
line  8  9 1  15.0 0 0
line  9 10 1  15.0 0 0
line 10 11 1  35.0 0 0
line 11 12 1  15.0 0 0
line 12 13 1  15.0 0 0
line 13 14 1  50.0 0 0
line 14 15 1 130.0 0 0
line 15 16 1  70.0 0 0
line 16 17 1  15.0 0 0
line 17  2 1  30.0 0 0
* feed to xfmr/generator
line 10 18 1 100.0 0 0
line 18 19 1 900.0 0 0
end

labelphase 0 G
labelphase 1 A

labelpole 1 surge
labelpole 2 ln1
* labelpole 3
labelpole 4 BrkrA
* labelpole 5
labelpole 6 ln2
* labelpole 7
labelpole 8 BrkrC
* labelpole 9
* labelpole 10
* labelpole 11
labelpole 12 BrkrD
* labelpole 13
labelpole 14 ln3
* labelpole 15
labelpole 16 BrkrB
* labelpole 17
* labelpole 18
labelpole 19 xfmr

* voltage output at breaker and xfmr locations, incoming surge
meter
pairs 1 0
poles 1 2 4 8 12 16 19

* other lines, easier to model here as resistors
resistor 355.0
pairs 1 0
poles 6 14

* source equivalent

resistor 19.0
pairs 1 0
poles 1

steepfront 78.95e3 1.36e-6 11.64e-6 1.0e-7 1.02
pairs 1 0
poles 1

* transformers and arresters

capacitor 2e-9
pairs 1 0
poles 19

* the 931 discharge is according to training manual, no
*  turnon conductance but includes lead inductance
* the 912 discharge model is using Cigre model
* arrbez 0.0 931.0e3 0.0 1.18e-6 4.0 1
arrbez 0.0 912.0e3 0.051 1.18e-6 4.0 1
pairs 1 0
poles 19

* insulators at breaker and xfmr locations

lpm 1800.0e3 551.3e3 1.831e-6
pairs 1 0
poles 4 8 12 16 19
