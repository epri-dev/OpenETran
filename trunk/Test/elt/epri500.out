  N   span     dT   Tmax
  1   0.00 6e-009 4e-005
Z-phase
  3.55000e+002
Z-modal
  3.55000e+002
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 6.
   #        vp        vm         i
   1 -371000.0 -371000.0   -1045.1
Power-frequency source at pole 14.
   #        vp        vm         i
   1 -371000.0 -371000.0   -1045.1
Power-frequency source at pole 1.
   #        vp        vm         i
   1 -371000.0 -371000.0  -19526.3
Meter at pole 1, from 1 to 0, max V   = 1.047516e+006
Meter at pole 2, from 1 to 0, max V   = 5.317161e+005
Meter at pole 4, from 1 to 0, max V   = 5.233453e+005
Meter at pole 8, from 1 to 0, max V   = 5.691786e+005
Meter at pole 12, from 1 to 0, max V   = 5.728369e+005
Meter at pole 16, from 1 to 0, max V   = 5.772754e+005
Meter at pole 19, from 1 to 0, max V   = 8.336524e+005
Meter at pole 19, arrester max I       = 1.068410e+003
insulator at pole 4, from 1 to 0 per-unit SI = 2.779222e-001
insulator at pole 8, from 1 to 0 per-unit SI = 2.926825e-001
insulator at pole 12, from 1 to 0 per-unit SI = 2.961182e-001
insulator at pole 16, from 1 to 0 per-unit SI = 2.840216e-001
insulator at pole 19, from 1 to 0 per-unit SI = 4.328866e-001
nr_iter = 15583, nr_max = 6
Execution time [sec]: 0.156
